pythontest.py

```diff
@@ -23,38 +26,43 @@ def formatcheck(file):
     # checking if the format is valid
     if file_extension not in allowed_extension:
         print('The following file has none supported file format:', file)
```

Comment:
Instead of just printing the error use the correct methods for error messages and exit the execution afterwards (see the google doc notes)